lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,62 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
+    collection.count{ |num| num.is_a? (Numeric)}
   end
 
   # Refer to: all?
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-all-3F
   def all_words_longer_than_length?(min_length)
+    collection.all? { |word| word.length > min_length}
   end
 
 
   # For the next 3 methods refer to: collect or map
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-any-3F
   def contains_a_word_longer_than?(min_length)
+     collection.any? { |word| word.length > min_length} # I don't understand my any? works. Any word?
```

Comment:
`any?` basically asks the question "are there any elements in this collection that fulfill the condition given in the block?"

The condition you give in the block is that the length of a word in the collection must be greater than the min_length number that is passed in. So if your collection is `['hi', 'bye', 'welcome']` and `min_length = 3`, then yes, there are some words in this collection that are longer than 3 characters ('welcome').